### ISSUE TITLE
We are unable to reproduce the performance described in the paper.

### ISSUE THREAD

**Issue by hao920293897** [HaochenLiu2000/QAP#1]
We are unable to reproduce the performance described in the paper with llama as our backbone. In the code, the gradients of llm are not required. I  change this part to LoRA fine-tuning, but the preformance is still very poor.

**Comment by HaochenLiu2000** [comment]
Our results are real. You can try more settings. Parts of the code can be flexibly revised.

**Comment by hao920293897** [comment]
I use LLAMA3.1-8b as the backbone. But the loss does not decrease, and accuracy remains between 25% and 30%.  The result appears to be a random selection. Are there some specific settings or training experience ？
